<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>switchWidgetInner</class>
 <widget class="QWidget" name="switchWidgetInner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>One</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Two</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Three</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Four</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>338</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ZsStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>320</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="toolTip">
       <string>Page_0</string>
      </property>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="toolTip">
       <string>Page_1</string>
      </property>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="toolTip">
       <string>Page_2</string>
      </property>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="toolTip">
       <string>Page_3</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZsStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>zsstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
